### ISSUE TITLE
not working with stdin

### ISSUE THREAD

**Issue by davegaulke** [logdyhq/logdy-core#67]
I’ve been trying out logdy, and am unable to get it to work when reading from stdin as in `journalctl -f -o json |logdy`. It does work when I use `logdy follow`. When using stdin, logdy immediately exits with no error messages. Running on x86_64 linux — almalinux 9.5.

**Comment by PeterOsinski** [comment]
Hey @davegaulke I just used it on my remote machine and everything works fine:
```
journalctl -f -o json | logdy --port=8443 -t --ui-ip=0.0.0.0
```
The `-t` option prints the content piped to logdy in the terminal. Is there any meaningful info printed to the terminal?

**Comment by MatteoGauthier** [comment]
Same issue
```
Linux 5.15.0-25-generic #25-Ubuntu SMP Wed Mar 30 15:54:22 UTC 2022 x86_64 x86_64 x86_64 GNU/Linux
```

**Comment by PeterOsinski** [comment]
I was able to replicate the issue with the latest version `0.13.2` for now the `journalctl -f -o json | logdy stdin` works as expected so please add `stdin` after the command for now. I'll take a look at the fix.

**Comment by PeterOsinski** [comment]
This has been fixed, the problem was with 0.13 release that removed starting web server from `logdy` command (without any command)
